edgepi-relay.html

```diff
@@ -1,137 +1,177 @@
+    <label for="node-input-transport">RPC Server:</label>
     <select id="node-input-transport">
       <option value="Local">Local</option>
       <option value="Network">Network</option>
     </select>
   </div>
-  <div class="form-row tcp" >
-    <input class ="tcp-address-input" type="text" id="node-input-tcpAddress" placeholder="IP Address">
+  <div class="form-row tcp">
+    <input
+      class="tcp-address-input"
+      type="text"
+      id="node-input-tcpAddress"
+      placeholder="IP Address/ Hostname"
+    />
     <label class="tcp-port-label" for="node-input-tcpPort">:</label>
-    <input class="tcp-port-input" type="text" id="node-input-tcpPort" placeholder="Port">
+    <input
+      class="tcp-port-input"
+      type="text"
+      id="node-input-tcpPort"
+      placeholder="Port"
+    />
   </div>
-  <div class="form-row energizeRelay">
-    <label for="node-input-energizeRelay">Energize:</label>
-    <input type="checkbox" name="radios" id="node-input-energizeRelay">
-    <p id="energizeStateText"></p>
+  <div class="form-row state">
+    <label for="node-input-state">State:</label>
+    <input
+      type="radio"
+      name="relayState"
+      id="node-input-closeRelay"
+      value="closeRelay"
+      checked
+    />
+    <label for="node-input-closeRelay">Closed</label>
+    <input
+      type="radio"
+      name="relayState"
+      id="node-input-openRelay"
+      value="openRelay"
+    />
+    <label for="node-input-openRelay">Open</label>
   </div>
 </script>
 
-
-<script type="text/html" data-help-name="edgepi-relay-node">
-	<p>EdgePi relay node that opens or closes relay.</p>
-	<h3>Properties</h3>
-	<dl class="message-properties">
-		<dt> RPC Server</dt>
-		<dd>The connection to your EdgePi's RPC Server. </dd>
-    <dt>Energize</dt>
-    <dd>Whether to open or close the relay on input.</dd>
-	</dl>
-	<h3>Inputs</h3>
-		<dl class="message-properties">
-			Any message can be used to trigger this node.
-		</dl>
-	<h3>Outputs</h3>
-		<dl class="message-properties">
-			<dt>payload <span class="property-type">string</span></dt>
-			<dd>A success message stating the relay status.</dd>
-		</dl>
-	<h3>References</h3>
-		<ul>
-			<li><a href="https://github.com/edgepi-cloud/node-red-edgepi-relay">GitHub</a> -the node's github repository </li>
-		</ul>
+<script type="text/html" data-help-name="relay">
+  <p>
+    Opens or closes the EdgePi relay. On the EdgePi, connect an input or supply voltage
+    to the COM (Common) port and connect the NO (Normally Open) port to the
+    device you want to control.
+  </p>
+  <h3>Properties</h3>
+  <dl class="message-properties">
+    <dt>RPC Server</dt>
+    <dd>The connection to your EdgePi's RPC Server.</dd>
+    <dt>State</dt>
+    <dd>
+      Configures input to trigger the relay to be in either an 'Open' or
+      'Closed' state.
+    </dd>
+  </dl>
+  <h3>Inputs</h3>
+  <dl class="message-properties">
+    Any message can be used to trigger this node.
+  </dl>
```

Comment:
we should change this. input = 1 should close the relay and 0 should open it.

Reply:
Good idea

Reply:
let's do this in a different PR.